bbs/CMakeLists.txt

```diff
@@ -152,3 +152,4 @@ add_library(bbs_lib ${COMMON_SOURCES} ${ZMODEM_SOURCES} ${PLATFORM_SOURCES})
 target_link_libraries(bbs_lib localui core sdk ${CL342_LIB} ${CMAKE_THREAD_LIBS_INIT})
 add_executable(bbs ${BBS_MAIN})
 target_link_libraries(bbs bbs_lib)
+target_link_libraries(bbs ssh)
```

Comment:
In file included from /home/rushfan/git/wwiv/bbs/ssh.cpp:19:0:
/home/rushfan/git/wwiv/bbs/../bbs/ssh.h:29:27: fatal error: libssh/libssh.h: No such file or directory
 #include <libssh/libssh.h>
                           ^
compilation terminated.
bbs/CMakeFiles/bbs_lib.dir/build.make:2006: recipe for target 'bbs/CMakeFiles/bbs_lib.dir/ssh.cpp.o' failed
make[2]: *** [bbs/CMakeFiles/bbs_lib.dir/ssh.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from /home/rushfan/git/wwiv/bbs/application.cpp:56:0:
/home/rushfan/git/wwiv/bbs/../bbs/ssh.h:29:27: fatal error: libssh/libssh.h: No such file or directory
 #include <libssh/libssh.h>

How did you include it? Did you copy it in somewhere?